README.md

```diff
@@ -0,0 +1,21 @@
+# CI Shortcuts
+
+This is a little go-based webserver that scrapes CircleCI build artifacts for a
+few well-known URLs that somebody might want quick access to.
+
+CircleCI's REST API is pretty crappy, so we do a lot of caching.  The caching
+is persisted in SQLite, and we use bun to work with it.
+
+## Server configuration
+
+The server runs in GCP under the name `ci-shortcuts`.  The VM runs with:
+
+ - No service account, so it has no access to GCP APIs
+ - A dedicated VPN, so it can't reach any other Determined instances on
+   internal IP addresses
```

Comment:
can we terraform this? i will add it if you want.

Reply:
idk, maybe we just wait for an infra person but having this stuff like not floating in the cloud would be nice

Reply:
who will keep the terraform state files?  That would be my only concern.

Reply:
Also, my guess is nobody has to update that instance... like ever

Reply:
update: this will run on the same node as https://det-ci.dzhu.dev